benefit/data/sheets/CaseEventToFields/CaseEventToFields.json

```diff
@@ -375,7 +375,7 @@
   },
   {
     "CaseEventID": "adjournCase",
-    "CaseFieldID": "adjournCaseMedicallyQualifiedPanelMemberName",
+    "CaseFieldID": "adjournCasePanelMember2",
```

Comment:
#############################
#############################
REVERT THIS BEFORE MERGING OR TESTING WITHOUT ADJOURNMENT FLAG
THIS IS FOR TESTING PURPOSES ONLY
#############################
#############################